### PR TITLE
Adopt list-builder-common

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: src/app/public/modules/list-view-grid/list-view-grid.component.ts [hunk 1]
```diff
@@ -35,17 +35,23 @@ import {
...
   ListState
 } from '@skyux/list-builder/modules/list/state';
 
+import {
+  ListItemModel
+} from '@skyux/list-builder-common/state/items/item.model';
+
+import {
+  ListSortFieldSelectorModel
+} from '@skyux/list-builder-common/state/sort/field-selector.model';
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/list-view-grid/list-view-grid.component.ts]: We can import from the barrel; exports are the only things that need the absolute path to the type.
```
import {
  getData,
  isObservable,
  ListItemModel,
  ListSortFieldSelectorModel
} from '@skyux/list-builder-common';
```

Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/list-view-grid/list-view-grid.component.ts]: There's a few other imports in the other files, too.

File: package.json [hunk 2]
```diff
@@ -30,12 +30,14 @@
...
+    "@skyux/list-builder-common": "^3.0.0-rc.1",
     "@skyux/modals": "^3.0.0-rc.0"
   },
   "dependencies": {},
   "devDependencies": {
     "@blackbaud/skyux": "2.30.0",
     "@blackbaud/skyux-builder": "1.28.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.5"
+    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.5",
+    "@skyux/list-builder-common": "^3.0.0-rc.1"
```
Comment (Blackbaud-SteveBrush) [inline package.json:41]: We'll need to release skyux2 with common, first, then remove this from the dev dependencies.

File: src/app/public/modules/list-view-grid/list-view-grid.component.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/list-view-grid/list-view-grid.component.ts]: And really, we're only writing those exports in this way for the skyux2 repo, to make sure that the specific files can be found.

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/list-view-grid/list-view-grid.component.ts]: fixed

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**